src/main/java/Tars.java

```diff
@@ -0,0 +1,81 @@
+import java.util.Scanner;
+
+public class Tars {
+    public static void main(String[] args) {
+        String logo = " ____        _        \n"
+                + "|  _ \\ _   _| | _____ \n"
+                + "| | | | | | | |/ / _ \\\n"
+                + "| |_| | |_| |   <  __/\n"
+                + "|____/ \\__,_|_|\\_\\___|\n";
+
+        // Opening speech.
+        System.out.println("    " + "------------------------------------------------------------");
+        System.out.println("    " + "Hello! I'm Tars.");
+        System.out.println("    " + "Ready to conquer the world? Or maybe just help with something smaller? What can I do for you?");
+        System.out.println("    " + "------------------------------------------------------------");
+
+        // Define an array to store tasks with a maximum capacity of 100 tasks, using the Task class to store data.
```

Comment:
the comments u have make the viewers understand the function of a method easily. good job